Server/tests/test_core_infrastructure_characterization.py

```
@@ -0,0 +1,1334 @@
+    return caplog
+
+
+@pytest.fixture
+def temp_telemetry_data():
+    """Fixture to provide temporary directory for telemetry data."""
+    with TemporaryDirectory() as tmpdir:
+        yield tmpdir
+
+
+@pytest.fixture
+def mock_telemetry_config(monkeypatch, temp_telemetry_data):
```

Ruff (0.14.14)

Unused function argument: `monkeypatch`

(ARG001)